SDKBuildScripts/mac_unity_RunAutoTests.sh

```diff
@@ -26,9 +28,9 @@ JenkernaughtSaveCloudScriptResults() {
 }
 
 RunMacJenkernaught() {
-    echo === Build OSX Client Target ===
+    echo === Build OSX Client Target === [I THOUGHT WE TOOK OUT _TC?! should we redirect?]
```

Comment:
Yes, TC should be gone, it should redirect to (depot)/sdks/${SdkName}/ExampleTestProject
I actually did that for you here:
https://github.com/PlayFab/SDKGenerator/tree/pg/UnityMac